console/src/assets/i18n/de.json

```
@@ -52,7 +52,7 @@
             "DETAIL": "Detail",
             "CREATE": "Erstellen",
             "MY": "Meine Informationen",
-            "LOGINNAMES":"Loginnamen",
+            "LOGINNAMES":"Zugangsmethoden",
```

```suggestion
            "LOGINNAMES":"Benutzernamen",
```

@fgerschwiler do you have a better idea?

---

or is it `Loginname`?

---

@fgerschwiler would name it 'Loginnamen'